docs/variants/protectli_vp2420/recovery.md

```diff
@@ -1,7 +1,95 @@
 # Recovery
 
-Following documentation describes the process of recovering hardware from brick
-state with [RTE](../../../transparent-validation/rte/introduction/) and Dasharo
-open-source firmware.
+## Intro
 
-**This page is under construction**
+The following documentation describes the process of recovering hardware from
+the brick state with [RTE](../../transparent-validation/rte/introduction.md) and
+Dasharo open-source firmware.
+
+## Prerequisites
+
+* [Prepared RTE](../../transparent-validation/rte/v1.1.0/quick-start-guide.md)
+* SOIC-8 Pomona clip
+* 6 wire cables
```

Comment:
In that place we should mention, that they are female-female cables

Reply:
information added